sdk/state/close.go

```diff
@@ -0,0 +1,120 @@
+package state
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/stellar/experimental-payment-channels/sdk/txbuild"
+	"github.com/stellar/go/txnbuild"
+	"github.com/stellar/go/xdr"
+)
+
+// The steps for a channel coordinated close are as followed:
+// 1. Initiator or Responder submits latest declaration tx
+// (in steps 2-4 Initiator and Responder are interchangeable, as long as they alternate)
+// 2. Initiator calls ProposeCoordinatedClose
+// 3. Responder calls ConfirmCoordinatedClose
+// 4. Initiator calls ConfirmCoordinatedClose
+
+type CoordinatedClose struct {
+	observationPeriodTime      time.Duration
+	observationPeriodLedgerGap int64
+	closeSignatures            []xdr.DecoratedSignature
```

Comment:
The other objects we have for Payment, Open, keep all their fields exported so that they're inspectable, and serializable. We should do the same here too I think, at least for consistency, but also so we have the same benefits and pattern for use.

Reply:
https://github.com/stellar/experimental-payment-channels/pull/95/commits/7b8efe8cb40017fd90aca14577e7effca343155b